### DISCUSSION TITLE
Where should we publish on PyKale?

### DISCUSSION THREAD

**Discussion by bobturneruk** [pykale/pykale#203] [Q&A] [answered]
Want maximum impact and publicity.

Some initial suggestions:

- Nature Methods
- Nature Computational Science
- Computer Methods and Programs in Biomedicine

Nature journals may be a little slow?

**Comment by haipinglu** [comment]
@bobturneruk The CIKM reviews are actually quite encouraging. A resource paper may fit well at  Nature Computational Science
https://www.nature.com/natcomputsci/content

**Resource**
A Resource is typically used to report on and present a large dataset, a new tool, or a new framework of broad interest for the computational science community. The main text (excluding abstract, Methods, references and figure legends) is approximately 3,500 words. The abstract is typically 100–150 words and is unreferenced. Resources have no more than 6 display items (figures and/or tables). An introduction (without heading) is followed by sections headed Results, Discussion and Methods. The Results and Methods should be divided by topical subheadings; the Discussion does not contain subheadings. As a guideline, Resources allow up to 50 references.

Resources include received/accepted dates. They may be accompanied by Supplementary Information.

Nature Methods does not take tools as resources: https://www.nature.com/nmeth/content
**Resource**
A Resource presents a large data set (such as a comprehensive list of proteins in an organelle or tissue, a genome-wide antibody library, coordinated analysis of cells or reagents by several different laboratories) of broad utility, interest and significance to the community. The main text (excluding abstract, online Methods, references and figure legends) is approximately 3,000 words. The abstract is typically 100-150 words, unreferenced. Resources have no more than 6 display items (figures and/or tables). An introduction (without heading) is followed by sections headed Results, Discussion and online Methods. The Results and online Methods should be divided by topical subheadings; the Discussion does not contain subheadings. As a guideline, Resources allow up to 50 references.

Resources include received/accepted dates. They may be accompanied by supplementary information. Resources are peer reviewed.

**Reply by haipinglu** [reply]
Nature Computation Science editorial: [Fostering multidisciplinary collaborations](https://www.nature.com/articles/s43588-021-00123-x)

**Comment by bobturneruk** [comment]
I agree "Nature Computational Science" looks like a better fit than "Nature Methods".

**Comment by haipinglu** [comment]
Also consider to submit a preliminary version to IJCNN/WCCI2022 https://wcci2022.org/
Paper submission: January 31, 2022
Notification of acceptance: April 26, 2022

Good timing and also good for an improved version for a top journal after collecting impacts.

**Comment by xianyuanliu** [comment, marked as answer]
We have secured a publication in CIKM.
